test/deprecated/IO/isclosed.chpl

```diff
@@ -0,0 +1,12 @@
+use IO;
+
+config const fileName = "test.txt";
```

Comment:
I think you'll want to add two things to do with the "test.txt" file:
- an `isclosed.cleanfiles` that lists it, so that start_test gets rid of it between runs
- an entry for it in the top level .gitignore so that we don't accidentally add it to later PRs